README.md

```diff
@@ -36,6 +36,8 @@ MONAI Label supports following OS with **GPU/CUDA** enabled.
 
 ### Development Release
 
+> **MONAI Label requires PyTorch version 1.10.0 or newer**
```

Comment:
this can go line after first statement.. under Installation

## Installation
MONAI Label supports following OS with **GPU/CUDA** enabled.
**MONAI Label requires PyTorch version 1.10.0 or newer.**

Reply:
Agree